docker/resources/build_fuzzy_libmath_dockerfile.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+DOCKERFILE=Dockerfile.mcalibmath
+
+generate_docker() {
+    
+cat > ${DOCKERFILE} <<HERE
+# Multistage build
+
+# Not the base image, will be used to copy stuff from
+FROM verificarlo/fuzzy:v0.4.1-lapack as fuzzy
```

Comment:
You actually don't need to do multistage to do `COPY --from`, I believe. So this could be removed and then the (current) lines 18 and 19 would reference the full image name.